provider/provider.go

```diff
@@ -1010,7 +1010,7 @@ func resourceShorelineObject(configJsStr string, key string) *schema.Resource {
 						return true
 					}
 				default:
```

Comment:
I suppose the above cases could also fail if `defowlt` will be `nil`. We should take in consideration that the condition may need to be more general, `if defowlt == nil return false`.

For now this seems a proper fix specifically for notebooks.